torch/nn/quantized/modules/_wrapped_modules.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import torch
+from torch.nn.modules import Module
```

Comment:
nit: torch.nn import Module?

Reply:
I think, that potentially can cause circular deps -- lemme check if I can